src/e2e/java/teammates/e2e/pageobjects/AppPage.java

```diff
@@ -212,7 +212,7 @@ public void verifyUnclickable(WebElement element) {
      */
     public void waitForConfirmationModalAndClickOk() {
         waitForModalShown();
-        WebElement okayButton = browser.driver.findElement(By.className("modal-btn-ok"));
+        WebElement okayButton = browser.driver.findElement(By.className("btn-info"));
```

Comment:
It looks like this line probably shouldn't be changed, it's responsible for the failure of some other E2e tests that depend on finding `modal-btn-ok` as well